src/sentry/integrations/issue_alert_image_builder.py

```diff
@@ -0,0 +1,121 @@
+import logging
+
+import sentry_sdk
+
+from sentry.api import client
+from sentry.charts import backend as charts
+from sentry.charts.types import ChartType
+from sentry.integrations.slack.message_builder.time_utils import (
+    get_approx_start_time,
+    get_relative_time,
+)
+from sentry.issues.grouptype import (
+    PerformanceP95EndpointRegressionGroupType,
+    ProfileFunctionRegressionType,
+)
+from sentry.models.apikey import ApiKey
+from sentry.models.group import Group
+from sentry.snuba.referrer import Referrer
+from sentry.types.integrations import ExternalProviderEnum
+from sentry.utils import metrics
+from sentry.utils.performance_issues.detectors.utils import escape_transaction
+
+
+class IssueAlertImageBuilder:
+    def __init__(
+        self, group: Group, provider: ExternalProviderEnum, logger: logging.Logger
```

Comment:
Can we have a default logger so calling side doesn't have to make one?

Reply:
We can. I was thinking email might want their own logger vs. slack. But the error would be the same so what you're suggesting makes more sense 😅

Reply:
Oh no, sorry, I meant just add a default for the argument, but you can keep the parameter!

So like
```
def __init__(
        self, group: Group, provider: ExternalProviderEnum, logger: logging.Logger = _DEFAULT_LOGGER
```

Reply:
What you had makes sense, allowing the caller to pass in their own logger if they choose